internal/store/pod.go

```diff
@@ -31,10 +32,11 @@ import (
 )
 
 const nodeUnreachablePodReason = "NodeLost"
+const postStartHookError = "PostStartHookError"
 
 var (
 	descPodLabelsDefaultLabels = []string{"namespace", "pod"}
-	containerWaitingReasons    = []string{"ContainerCreating", "CrashLoopBackOff", "CreateContainerConfigError", "ErrImagePull", "ImagePullBackOff", "CreateContainerError", "InvalidImageName"}
+	containerWaitingReasons    = []string{"ContainerCreating", "CrashLoopBackOff", "CreateContainerConfigError", "ErrImagePull", "ImagePullBackOff", "CreateContainerError", "InvalidImageName", postStartHookError}
```

Comment:
Let's not have a variable just for this reason string.
```suggestion
	containerWaitingReasons    = []string{"ContainerCreating", "CrashLoopBackOff", "CreateContainerConfigError", "ErrImagePull", "ImagePullBackOff", "CreateContainerError", "InvalidImageName", "PostStartHookError"}
```